app/views/home/dashboard.html.erb

```diff
@@ -10,21 +10,21 @@
   <div class="column-one-third">
     <div class="data">
       <span class="data-item bold-xlarge"><%= number_with_delimiter(@datasets_count) %></span>
-      <span class="data-item bold-xsmall">datasets</span>
+      <span class="data-item bold-xsmall">total datasets</span>
     </div>
   </div>
 
   <div class="column-one-third">
     <div class="data">
       <span class="data-item bold-xlarge"><%= number_with_delimiter(@published_datasets_count) %></span>
-      <span class="data-item bold-xsmall">published datasets</span>
+      <span class="data-item bold-xsmall">datasets published with datafiles</span>
     </div>
   </div>
 
   <div class="column-one-third">
     <div class="data">
       <span class="data-item bold-xlarge"><%= number_with_delimiter(@unpublished_datasets_count) %></span>
-      <span class="data-item bold-xsmall">datasets with no datafiles</span>
+      <span class="data-item bold-xsmall">datasets published with no datafiles</span>
```

Comment:
This is confusing. The variable says `@unpublished_datasets_` but the suffix says "datasets published".

Reply:
I agree, "unpublished" is an awful name for a dataset published with no datafile, but apparently it's what everyone calls it in the legacy system. ** sigh **

I am happy to just say "published with no datafiles" everywhere. I'll make the change.

Reply:
The variable name implies it is a count of datasets that have not been published. However, the suffix says "datasets published".

Reply:
Cool! I guess we should try and introduce a domain language that's easier to understand in this brave new world :)